### DISCUSSION TITLE
Запуск от имени пользователя

### DISCUSSION THREAD

**Discussion by Shelushun** [Storik4pro/goodbyeDPI-UI#41] [Ideas]
Есть возможность разрешить запускать goodbyeDPI.exe с правами пользователя или хотя бы Invoker? Или запускать как сервис?
Пробовал разные ключи запуска через командную строку, но не подобрал. Можно добавить в релиз?

**Comment by Storik4pro** [comment]
Без прав администратора у программы будет ограниченный функционал. Я могу отключить проверку на наличие прав запуска, но  это может вызвать большое количество ошибок. Начиная от невозможности добавить приложение в автозапуск, заканчивая невозможностью использования zapret/goodbyeDPI

**Comment by Shelushun** [comment]
Есть тогда возможность реализовать возможность запуска через сервис? Ему можно и админские права дать.
Так же во время использования заметили проблему при сверке итогов у эквайрингового терминала подключенного к компьютеру, не удавалось ему выполнить обмен с банком. Только после выгрузки запрета пошел обмен. Но вы вряд ли будете этим заниматься :-).

**Reply by Storik4pro** [reply]
> Есть тогда возможность реализовать возможность запуска через сервис?

Не совсем понимаю. Вы хотите, чтобы была кнопка "Установить zapret как сервис", которая будет устанавливать компонент как сервис отдельно от UI?
> Так же во время использования заметили проблему при сверке итогов у эквайрингового терминала подключенного к компьютеру,

Эта проблема связана с используемым пресетом

**Reply by Shelushun** [reply]
> Не совсем понимаю. Вы хотите, чтобы была кнопка "Установить zapret как сервис", которая будет устанавливать компонент как сервис отдельно от UI?

Да всё верно. Это бы решило проблему автозапуска на компьютерах с ограниченными правами пользователя.



> Эта проблема связана с используемым пресетом

Да понимаю всё. Надеюсь решим этот вопрос самостоятельно.

**Reply by Storik4pro** [reply]
> Да всё верно. Это бы решило проблему автозапуска на компьютерах с ограниченными правами пользователя.

Попробую реализовать подобное

**Reply by Shelushun** [reply]
Это будет просто супер круто!
